presto-accumulo/src/main/java/io/prestosql/plugin/accumulo/conf/AccumuloSessionProperties.java

```diff
@@ -158,4 +158,17 @@ public static boolean isIndexShortCircuitEnabled(ConnectorSession session)
     {
         return session.getProperty(INDEX_SHORT_CIRCUIT_CARDINALITY_FETCH, Boolean.class);
     }
+
+    private static PropertyMetadata<Duration> durationProperty(String name, String description, Duration defaultValue, boolean hidden)
```

Comment:
I thought about that, but it seemed wrong to add a dependency for a single utility method, plus it leads towards plugin toolkit becoming a "junk drawer" which we avoid (that's why we only have modules with focused names and not "util").

* For Blackhole, it adds a number of dependencies like Guice and `configuration` that the plugin isn't already using (no need for `Bootstrap` since the plugin has no configuration).
* For Accumulo, it only adds `parameternames`, so that seems ok.
* We also need it in `SystemSessionProperty`, which is in `presto-main` and won't depend on the plugin toolkit.

However, I think it is reasonable to add it to the toolkit and save a bit of duplication for the modules that already depend on it.

Reply:
I thought about that, but it seemed wrong to add a dependency for a single utility method, plus it leads towards plugin toolkit becoming a "junk drawer" which we avoid (that's why we only have modules with focused names and not "util").

* Hive, Iceberg, and Raptor only use `dataSizeProperty`.
* For Blackhole, it adds a number of dependencies like Guice and `configuration` that the plugin isn't already using (no need for `Bootstrap` since the plugin has no configuration).
* For Accumulo, it only adds `parameternames`, so that seems ok.
* We also need it in `SystemSessionProperty`, which is in `presto-main` and won't depend on the plugin toolkit.

Given this, I'd rather leave a few copies of this simple method.